src/Control/Effect/Resumable.hs

```diff
@@ -22,8 +32,13 @@ instance Effect (Resumable err) where
   handle state handler (Resumable err k) = Resumable err (handler . (<$ state) . k)
 
 -- | Throw an error which can be resumed with a value of its result type.
+-- Note that the type parameters in the @err a@ paramater and @m a@ parameter must match
+-- up; this is so that the calling context knows what type of value this computation
+-- expected in the success case.
```

Comment:
1. Suggest: “…; this indicates the type that the error must be resumed with.”

2. Soft wrap, please 🙏